### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by hazel-ocean** [helix-editor/helix#5886] [Show and tell]
Just wanted to pop in and say "thank you!" to the community for this project which I use literally every day. It's really wonderful.
And also a thank you to the folks who made theme inheritance a thing so that I can now [create dynamic overrides for themes](https://github.com/davidarmstronglewis/nix-config/commit/5210422bb8968063457e004d5e79ec228e947b40#diff-b0d26954aacde57367aaf21a466afaad215d6821326258b73640374d537dfdb4R46) I want to use regularly but my OCD prevented me from using a theme whose background did not match the background of my terminal of choice (Alacritty).

Hope you all have a fantastic week <3

**Comment by CptPotato** [comment]
> [create dynamic overrides for themes](https://github.com/davidarmstronglewis/nix-config/commit/5210422bb8968063457e004d5e79ec228e947b40#diff-b0d26954aacde57367aaf21a466afaad215d6821326258b73640374d537dfdb4R46) I want to use regularly but my OCD prevented me from using a theme whose background did not match the background of my terminal [...]

If I understand #5839 correctly, with this PR you'd be able to only set a single `"ui.background" = {}` override in your editor config ;)
